website/docs/r/frontdoor.html.markdown

```diff
@@ -70,6 +70,89 @@ resource "azurerm_frontdoor" "example" {
 }
 ```
 
+Custom https configurations can also be defined as a separate resource. This allows for parallel creation/update.
```

Comment:
shouldn't this be in a separate file?
looking at:

https://www.terraform.io/docs/providers/azurerm/r/key_vault.html
https://www.terraform.io/docs/providers/azurerm/r/key_vault_access_policy.html

Reply:
👍